src/dev-box/dev-box-k8s-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Could you deploy dev-box as a 'POD'?
I'm not sure why it should be a daemonset.

Reply:
Sure, it could be, need some test to test it
 use daemonset is consist to yundong's previous deployment, most of deployed service as daemonset.